esrally/track/params.py

```diff
@@ -732,6 +738,48 @@ def percent_completed(self):
         return self.current_bulk / self.total_bulks
 
 
+class OpenPointInTimeParamSource(ParamSource):
+    def __init__(self, track, params, **kwargs):
+        super().__init__(track, params, **kwargs)
+        if len(track.indices) == 1:
+            default_target = track.indices[0].name
+        elif len(track.data_streams) == 1:
+            default_target = track.data_streams[0].name
+        else:
+            default_target = None
```

Comment:
all this default_target stuff is copied directly from the QueryParamSource to support a default index selection.  it doesn't need to be here, or else it can be extracted to its own function, if desired